webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/healthcheck/HealthcheckService.scala

```diff
@@ -59,4 +65,63 @@ final object CheckCoreNumber extends Check {
       case n => Ok(s"${n} cores available")
     }
   }
+}
+
+final object CheckFreeSpace extends Check {
+  def name: CheckName = CheckName("Disk free space available")
+
+  final case class SpaceInfo(val partition: String, val free: Long, val available: Long){
+    def pourcentage: Double =
+      if (available != 0) {
+        (free * 100 / BigDecimal(available)).setScale(2, BigDecimal.RoundingMode.HALF_UP).toDouble
```

Comment:
not sur why the BigDecimal here? `BigDecimal(free * 100)` if you want to avoid an overflow on the mult, but why on the denominator?